### DISCUSSION TITLE
getting an error regarding ffmpeg

### DISCUSSION THREAD

**Discussion by magemcgee** [hacksider/Deep-Live-Cam#327] [Q&A]
[ERROR:0@0.001] global cap_ffmpeg_impl.hpp:1237 open Could not find decoder for codec_id=61
[ERROR:0@0.001] global cap_ffmpeg_impl.hpp:1286 open VIDEOIO/FFMPEG: Failed to initialize VideoCapture

aside from this, the program also crashes anytime i try to export anything at all. live camera works totally fine but cant do anything besides that.

**Comment by secmanhck** [comment]
1. You need to install ffmpeg or download ffmpeg unzip and paste your C:\Program Files.
2. Go to C:\Program Files\ffmpeg\bin copy this 
3. Go to Edid system click on path 
4. Add new path 
5. Save and ok 
6. Go to your  cmd and now type ffmpeg  
 
-------------------ENJOY-------------
